app/controllers/SignInController.scala

```diff
@@ -42,6 +43,17 @@ abstract class SignInController(val applicationClient: ApplicationClient, cacheC
       }
   }
 
+  def sdipPresent = CSRUserAwareAction { implicit request =>
```

Comment:
What do we present exactly? sdip is "consider me as sdip? Sdip alone maybe confusion as we have sdip and also sdip from faststream.